string pid
float32 burst_time
uint32 prio
